src/class.ziprecipes.php

```diff
@@ -914,6 +995,14 @@ public static function zrdn_iframe_content($post_info = null, $get_info = null)
 
 		require_once(ABSPATH . 'wp-admin/includes/plugin.php');
 		$settings_page_url = admin_url( 'admin.php?page=' . 'zrdn-settings' );
+		$registration_url = admin_url( 'admin.php?page=' . 'zrdn-register' );
+
+        $skip_reg=false;
+        if (!$skip_reg) {
+            if (isset($_COOKIE['skip-registration'])) {
+                $skip_reg = true;
+            }
+        }
 
 		Util::print_view('create-update-recipe', array(
```

Comment:
I think it's better to show `register.twig` here in the cases where we need to show it. This way we don't have to pollute `create-update-recipe` with too much registration stuff and we can remove macros from `register.twig`